conjur/policy/policy_logic.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+"""
+PolicyLogic module
+
+This module is the business logic for executing the POLICY command
+"""
+import json
+
+
+class PolicyLogic:
```

Comment:
Maybe PolicyLogic and VariableLogic should inherit from same abstract class.

Reply:
agreed let's talk about this design

Reply:
you ca look here https://docs.python.org/3/library/abc.html for implement more strong abstraction

Reply:
BTW all logic classes has the same interface am I right? so we can inherit from the same ILogic interface. this is not for now but can help us with Dekel builder pattern

Reply:
Sound good @mbenita-Cyberark 
what is dekel builder pattern?

Reply:
not necessarily. User and Host are different than Policy is different then Init etc